About.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js" defer></script>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <section class="first-navbar">
+            <nav class="nav1">
+                <ul class="list l1">
+                    <li><a href=""><img src="./images/facebook.png" alt="fb-logo">
+                        </a></li>
+                    <li><a href=""><img src="./images/twitter.png" alt="twitter"></a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Logout</a></li>
+                </ul>
+            </nav>
+        </section>
+    
+        <section class="second-navbar">
+            <div class="logo-img"><a href="./index.html"><img src="./images/cannewlogo2018.png" alt=""></a></div>
+            <nav class="nav2">
+                <ul class="list l2">
+                    <li><a href="./About.html" class="about">About</a></li>
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Sponsor</a></li>
+                    <li><a href="">News</a></li>
+                    <li><span><a href="">CC&nbsp;Campaign</a></span></li>
+                </ul>
+            </nav>
+        </section>
+        <section class="section3">
+            <button class="hamburger">
+                <div class="bar"></div>
+            </button>
+            <nav class="mobile-nav">
+                <ul>
+                    <li><a href="./index.html">Home</a></li>
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Sponsor</a></li>
+                    <li><a href="">News</a></li>
+                    <li><span><a href="">CC&nbsp;Campaign</a></span></li>
+                </ul>
+            </nav>
+        </section>
+    </header>
```

Comment:
- [ ] As mentioned by the previous reviewer. You need to work on the navbar top and bottom alignment. You've not attended to this request. Kindly look into it and fix.